tools/xml/csv2xml.py

```diff
@@ -15,7 +15,7 @@
 #   may be used to endorse or promote products derived from this work without
 #   specific prior written permission.
 #
-# THIS SOFTWARE IS PROVIDED BY INTEL CORPORATION "AS IS"
```

Comment:
I got it from stability tests: https://github.com/crosswalk-project/crosswalk-test-suite/blob/master/stability/stability-iterative-android-tests/iterative/Install_Uninstall_Repeatedly.py